code/src/components/product-card/ProductCard.scss

```diff
@@ -38,4 +49,27 @@
     @include transition((all, "medium"));
     backdrop-filter: blur(spacing(2));
   }
+
+  &__footer {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: auto;
+    padding-top: spacing(2);
+    border-top: 1px solid var(--spa-products-container-border-color, $light);
+  }
+
+  &__button {
+    border: 1px solid var(--spa-best-sellers-price-color, $black);
+    background-color: transparent;
```

Comment:
Maybe we have this color ins design system. Use it instead

Reply:
> Maybe we have this color ins design system. Use it instead

already exist variable for `transparent`
fixed